{{cookiecutter.project_slug}}/backend/Dockerfile

```diff
@@ -104,3 +104,6 @@ COPY --from=build /app/ /app/
 COPY config/ /app/src/config/
 COPY manage.py /app/src/
 COPY {{ cookiecutter.project_slug }}/ /app/src/{{ cookiecutter.project_slug }}/
```

Comment:
The `COPY` commands required a `--chown=django:django` which in turn requires the django user to be added before `COPY`